engines/bastion_katello/app/assets/javascripts/bastion_katello/sync-plans/details/sync-plan-details-info.controller.js

```diff
@@ -26,8 +26,12 @@ angular.module('Bastion.sync-plans').controller('SyncPlanDetailsInfoController',
             $scope.panel = $scope.panel || {loading: false};
 
             function updateSyncPlan(syncPlan) {
-                syncPlan.syncDate = new Date(syncPlan['sync_date']);
-                syncPlan.syncTime = new Date(syncPlan['sync_date']);
+                var syncDate = syncPlan['sync_date'];
+                if (_.isNaN(Date.parse(syncDate))) {
```

Comment:
Oh I see, this is the fix.  I wonder if a comment would be in order here because otherwise I may have no idea why this line is here and remove it.